From d0c41a08fba482cd0fe79acc93e1dacd7be74025 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 4 Aug 2016 15:13:11 -0700
Subject: [PATCH] TOOLS-1508 prep_dataset_in_jpc.sh using wrong "mantapath" dir
 for built image export Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>

---
 tools/prep_dataset_in_jpc.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/prep_dataset_in_jpc.sh b/tools/prep_dataset_in_jpc.sh
index afc1332..6d2c5d9 100755
--- a/tools/prep_dataset_in_jpc.sh
+++ b/tools/prep_dataset_in_jpc.sh
@@ -362,7 +362,7 @@ if [[ -n ${SDC_LOCAL_BUILD} ]]; then
     curl -sS -f -o ${output_dir}/${image_filename} ${SDC_IMGAPI_URL}/images/${image_id}/file
 
 else
-    mantapath=/${SDC_ACCOUNT}${MG_OUT_PATH}/${build_name}/$(echo ${image_version} | cut -d '-' -f1,2)/${build_name}
+    mantapath=/${SDC_ACCOUNT}${MG_OUT_PATH}/${build_name}/$(echo ${image_version} | sed -E 's/-g[a-f0-9]{7,40}$//')/${build_name}
     mmkdir -p ${mantapath}
 
     manta_bits=/tmp/manta-exported-image.$$
-- 
2.21.0

